roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -391,15 +419,10 @@ spec:
 {% endif %}
             - name: policysync
               mountPath: /var/run/nodeagent
-{% if calico_bpf_enabled %}
             # For eBPF mode, we need to be able to mount the BPF filesystem at /sys/fs/bpf so we mount in the
             # parent directory.
-            - name: sysfs
-              mountPath: /sys/fs/
-              # Bidirectional means that, if we mount the BPF filesystem at /sys/fs/bpf it will propagate to the host.
-              # If the host is known to mount that filesystem already then Bidirectional can be omitted.
-              mountPropagation: Bidirectional
-{% endif %}
```

Comment:
I would like to know why `calico_bpf_enabled` was removed.

Reply:
It's in the default installation mode of calico, and I think we should follow it here. 
See https://github.com/projectcalico/calico/blob/v3.28.1/manifests/calico.yaml#L4866-L4888

And on the other hand,  the advantage of this is that if one want to enable calico-bpf, they need only change `FelixConfiguration` to open `bpfEnabled` and no need to update calico-node.

Reply:
Ok, So it seems that the remaining differences in their `bpf enabled` setup are `FELIX_BPFENABLED`, `CALICO_IPV4POOL_IPIP`, `kubernetes_service_host` and `kubernetes_service_port`.